### PR TITLE
apm-1147: create role, plays and pipeline to support tearing down old…

### PR DESCRIPTION
**Pull request by vurt007**
…er PR branches

### CODE REVIEW

File: ansible/roles/destroy-api-deployment-dns-entry/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 
+- name: print workspace
+  debug: msg="workspace - {{ workspace }}"
```
Comment (Sourceless) [inline ansible/roles/destroy-api-deployment-dns-entry/tasks/main.yml:3]: Is this still required?

File: ansible/roles/destroy-ecs-proxies/tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 
+- name: print workspace
+  debug: msg="workspace - {{ workspace }}"
```
Comment (Sourceless) [inline ansible/roles/destroy-ecs-proxies/tasks/main.yml:3]: likewise here

File: ansible/roles/destroy-api-deployment-dns-entry/tasks/main.yml (hunk 1 above)
Comment (vurt007) [reply ansible/roles/destroy-api-deployment-dns-entry/tasks/main.yml:3]: yes .. surely the name 'print workspace' is comment enough on these ;)?

File: ansible/roles/destroy-ecs-proxies/tasks/main.yml (hunk 2 above)
Comment (vurt007) [reply ansible/roles/destroy-ecs-proxies/tasks/main.yml:3]: ditto

**Pull request merged**